service/src/main/java/bio/terra/pipelines/app/controller/PipelinesApiController.java

```diff
@@ -48,4 +58,11 @@ static ApiPipelinesGetResult pipelinesToApi(List<Pipeline> pipelineList) {
 
     return apiResult;
   }
+
+  static ApiPipeline pipelineToApi(Pipeline pipelineInfo) {
```

Comment:
yeah! I didn't do that originally because I think when we remove the toy example we won't need this anymore, but it's nice to have it in its own function anyway.